core/src/test/java/org/arquillian/smart/testing/configuration/ConfigurationTest.java

```diff
@@ -15,6 +20,12 @@
 
 public class ConfigurationTest {
 
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    @Rule
```

Comment:
Use assertJ assertions instead of this rule http://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#exception-assertion